spec/services/rapid_ready_for_decision/hypertension_observation_data_spec.rb

```diff
@@ -16,6 +16,7 @@
   let(:response_with_recent_bp) do
     original_first_bp_reading = response.body['entry'].first
     original_first_bp_reading['resource']['issued'] = (Time.zone.today - 2.weeks).to_s
+    original_first_bp_reading['resource']['effectiveDateTime'] = (Time.zone.today - 2.weeks).to_s
```

Comment:
I... don't love this but it gets the job done. I know in some frameworks there's a more robust system for generating mock data, and I'm sure that exists in Rails land somewhere.